docs/src/content/pages/reader-api.mdoc

```diff
@@ -28,10 +28,10 @@ You can then create a new `reader` by calling `createReader` and passing it two
 1. The path (relative to root) to the content files (as specified in the `directory` property of a given collection/singleton)
```

Comment:
Now that I changed the first argument to `process.cwd()` I just wanted to check that this line is still correct? 🤔

Reply:
It is still technically correct, perhaps we can add more to it to simplify it?

```suggestion
1. The path (relative to root) to the content files as specified in the `directory` property of a given collection/singleton. In most setups this will just be the current working directory: `process.cwd()`.
```